JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -482,6 +482,10 @@ - (UICollectionViewCell *)collectionView:(JSQMessagesCollectionView *)collection
             cell.messageBubbleImageView.image = [bubbleImageDataSource messageBubbleImage];
             cell.messageBubbleImageView.highlightedImage = [bubbleImageDataSource messageBubbleHighlightedImage];
         }
+        else {
+            cell.messageBubbleImageView.image = nil;
+            cell.messageBubbleImageView.highlightedImage = nil;
+        }
```

Comment:
Let's simplify, just remove the first if:

````objective-c
id<JSQMessageBubbleImageDataSource> bubbleImageDataSource = [collectionView.dataSource collectionView:collectionView messageBubbleImageDataForItemAtIndexPath:indexPath];
cell.messageBubbleImageView.image = [bubbleImageDataSource messageBubbleImage];
cell.messageBubbleImageView.highlightedImage = [bubbleImageDataSource messageBubbleHighlightedImage];
````

this has the same effect, if you return `nil`, we set `nil`